x-pack/legacy/plugins/uptime/public/components/functional/monitor_status_bar.tsx

```diff
@@ -93,11 +102,29 @@ export const MonitorStatusBarComponent = ({ data, monitorId }: Props) => {
                 defaultMessage: 'Time since last check',
               }
             )}
-            grow={false}
+            grow={true}
           >
             {moment(new Date(timestamp).valueOf()).fromNow()}
           </EuiFlexItem>
         </EuiFlexGroup>
+        {certificateValidity && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiText
+              color="subdued"
+              grow={false}
+              size="s"
+              aria-label={i18n.translate('xpack.uptime.monitorStatusBar.sslCertificateExpiry', {
```

Comment:
Do you think we should supply a separate ID for the `aria-label` vs the component's content, despite them being the same string?

A key like `xpack.uptime.monitorStatusBar.sslCertificateExpiry.ariaLabel` and `xpack.uptime.monitorStatusBar.sslCertificateExpiry.content` might be appropriate.

Reply:
yes make sense, done !!